src/routes/blog/+page.svelte

```diff
@@ -0,0 +1,261 @@
+<script lang="ts">
+  import type { PageData } from './$types';
+  import { Temporal } from '@js-temporal/polyfill';
+  import { readingTime } from '$lib/public/blog/utils';
+  import Spacing from '$lib/public/legacy/spacing.svelte';
+  import LabelField from './labelfield.svelte';
+  import Labels from './labels.svelte';
+
+  export let data: PageData;
+
+  async function filterPosts(event: CustomEvent) {
+    const query = event.detail.join(';') || 'all';
+
+    history.replaceState({}, '', location.pathname + (query === 'all' ? '?l=' + query : ''));
+
+    const target = new URL(`/blog/${query}.json`, location.origin);
+    const response = await fetch(target.toString());
+    const blogOutput = await response.json();
+    if (blogOutput.posts) data.posts = blogOutput.posts;
+  }
+</script>
+
+<svelte:head>
+  <title>README | ACM at CSUF</title>
+</svelte:head>
+
+<Spacing --min="175px" --med="200px" --max="200px" />
+
+<section class="main-header">
+  <img src="/assets/readme-logomark.svg" alt="README by acmCSUF" />
+
+  <div>
+    <h1 class="size-xxl">README</h1>
+    <h2 class="size-md">by ACM at <b class="acm-blue">CSUF</b></h2>
+  </div>
+</section>
+
+<section>
+  <h2 class="subtitle headers size-md">
+    The official ACM at CSUF blog.<a href="/blog.xml"
```

Comment:
Update link to RSS feed.
```suggestion
    The official ACM at CSUF blog.<a href="/feed.xml"
```

Reply:
Resolved by commit <https://github.com/EthanThatOneKid/acmcsuf.com/commit/a0c13956aed4c9bc4ca98a517e9af718f1489c70>.